docs/3-KnowledgeBase.stories.mdx

```diff
@@ -219,4 +220,176 @@ This would then be the result:
   <MyCustomElement />
 </ThemeProvider>
 
+## Form Handling
+
+This guide demonstrates the use of the `Form` component.
+
+### #1 Example with UI5 Web Components for React
+
+This Form is implemented with the `Form` component from UI5 Web Components for React.
+The children of the `Form` should only be `FormGroup` and `FormItem` in order to preserve the intended design. The children of `FormGroup` should only be `FormItem` and the children of `FormItem` can be an arbitrary React Node.
+
+You can then create a form with UI5 Web Components for React as follows. Also you can find this example in a Codesandbox. [![Edit wcr form](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/wcr-form-5f9c24?fontsize=14&hidenavigation=1&theme=dark)
+
+<details>
+    <summary>Show code</summary>
+
+```jsx
+import React, { useState } from 'react';
+import {
+  ThemeProvider,
+  Form,
+  FormGroup,
+  FormItem,
+  Input,
+  Option,
+  Select,
+  CheckBox,
+  InputType,
+  Button,
+  MultiComboBox,
+  MultiComboBoxItem,
+  DatePicker
+} from '@ui5/webcomponents-react';
+import '@ui5/webcomponents/dist/features/InputElementsFormSupport.js';
+
+export default function CreateAccountForm() {
+  const initialValues = {
```

Comment:
This will update each component that uses this state every time the state updates. Please use a single state for each value.